lib/src/firebase_app/components/FirebaseLoginView.tsx

```diff
@@ -148,6 +157,38 @@ export function FirebaseLoginView({
                     </div>;
             } else if (!ignoredCodes.includes(authController.authError.code)) {
                 console.error(authController.authError);
+                if (authController.authError.code == 'auth/multi-factor-auth-required') {
```

Comment:
I would rather have this logic in an independent function, not inside `buildErrorView`

Reply:
got it will do. you mean I should wrap the if/else statement in a separate function right? Triggering two factor SMS verification is done when catching the error according to google's example https://firebase.google.com/docs/auth/web/multi-factor#signing_users_in_with_a_second_factor

Reply:
Right, simply include the code inside the else if in a different function